CMakeLists.txt

```diff
@@ -19,7 +19,15 @@ include(clang)
 
 include(conan)
 
-conan_cmake_run(CONANFILE conanfile.txt BASIC_SETUP CMAKE_TARGETS BUILD poco missing)
+conan_cmake_run(REQUIRES doctest/2.3.7
```

Comment:
Is there a reason not to use BASIC_SETUP ?

Reply:
actually required now (for qt)